docs/QTIP.md

```diff
@@ -0,0 +1,25 @@
+# QTIP
+
+## What?
+
+QTIP is an MsQuic specific concept that allows MsQuic to exercise the QUIC protocol over XDP using TCP packets.
+
+## Why?
+
+Certain hardware / networks / cloud environments are optimized for TCP traffic. Instead of pushing for those environments to also optimize for UDP, QTIP allows us to leverage existing hardware with MsQuic for cheap.
+
+## How?
+
+
+To use QTIP, you first must be running XDP.
+
+QTIPEnabled in QUIC_SETTINGS controls whether client connections will send/recv packets over QTIP xor UDP and whether listeners will optionally accept QTIP traffic.
+
+Opt in to use QTIP for listeners by setting QTIPEnabled to true at a global level prior to starting the listener. Additionally, all client connections created thereafter will send/recv data over QTIP.
+This setting can be overrided per client connection, allowing you to create some connections that send QTIP or UDP traffic.
+
+Important:
```

Comment:
nit: Can we make this a title level 3 or 4, or make it bold, or using the github markdown tagged box?
It isn't very visible once you go to the pretty view.

```
> [!IMPORTANT]  
> Crucial information necessary for users to succeed.
```